### PR TITLE
Update adoptium links

### PR DESCRIPTION
**Pull request by wipfli**
Expands adoptium java install links to directly point to win, mac, and linux installers.﻿

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Pull requests are welcome! To set up your development environment:
 
 - Fork the repo
-- [Install Java 16 or later](https://adoptium.net/installation.html)
+- Install Java 16 or later. You can download Java manually from [Adoptium](https://adoptium.net/installation.html) or use their installers for [Windows](https://adoptium.net/installation.html#windows-msi), [macOS](https://adoptium.net/installation.html#macos-pkg), and [Linux](https://github.com/adoptium/website-v2/blob/main/src/asciidoc-pages/installation/linux.adoc).
```
Comment (msbarry) [inline CONTRIBUTING.md:6]: This install url is also referenced from planetiler-examples/README.md and README.md https://github.com/onthegomap/planetiler/search?q=https%3A%2F%2Fadoptium.net%2Finstallation.html - think we should change it there as well?

Comment (wipfli) [reply CONTRIBUTING.md:6]: Good catch. I suggest cross-referencing the CONTRIBUTING.md file, see commit

Comment (msbarry) [reply CONTRIBUTING.md:6]: Thanks! that looks good. What do you think about inlining the command for common installers?
- `brew install --cask temurin`
- `apt-get install openjdk-17-jdk` (probably better to prefer the recommended distro's openjdk build)

Comment (wipfli) [reply CONTRIBUTING.md:6]: Sounds like a good idea

Comment (msbarry) [reply CONTRIBUTING.md:6]: Cool, do you want to add those or want me to?

Comment (msbarry) [reply CONTRIBUTING.md]: Looks like I can update your branch - how does this look?

**Review by wipfli — COMMENTED** [review]
Looks good to me!

**Pull request merged**